ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -509,6 +530,19 @@ def process_cluster_cli_params(config):
     ocs_registry_image = get_cli_param(config, f"ocs_registry_image{suffix}")
     if ocs_registry_image:
         ocsci_config.DEPLOYMENT["ocs_registry_image"] = ocs_registry_image
+    install_lvmo = get_cli_param(config, "install_lvmo")
+    if install_lvmo:
+        ocsci_config.DEPLOYMENT["install_lvmo"] = True
+        number_of_disks = get_cli_param(config, "lvmo_disks")
+        if number_of_disks is None:
+            ocsci_config.DEPLOYMENT["lvmo_disks"] = 3
```

Comment:
why is it set to 3?
can we make this dynamic and inject the number from a conf file?

Reply:
This is dynamic. Only if not configured via --lvmo-disks <number> or didn't pass a yaml with:
DEPLOYMENT:
  lvmo_disks: <number>